concepticondata/conceptlists/Mountain-1978-267.tsv

```diff
@@ -0,0 +1,268 @@
+Mountain-1978-267-206	206	cuerno	horn	269	1393	HORN (ANATOMY)
+Mountain-1978-267-207	207	pluma	feather	270	1201	FEATHER
+Mountain-1978-267-208	208	uña	claw	271	72	CLAW
+Mountain-1978-267-209	209	barriga	belly	272	1251	BELLY
+Mountain-1978-267-210	210	hígado	liver	273	1224	LIVER
+Mountain-1978-267-211	211	beber	to drink	274	1401	DRINK
+Mountain-1978-267-212	212	comer	to eat	275	1336	EAT
+Mountain-1978-267-213	213	morder	to bite	276	1403	BITE
+Mountain-1978-267-214	214	ver	to see	277	1409	SEE
+Mountain-1978-267-215	215	oír	to hear	278	1408	HEAR
+Mountain-1978-267-216	216	saber	to know	279	1410	KNOW (SOMETHING)
+Mountain-1978-267-217	217	dormir	to sleep	280	1585	SLEEP
+Mountain-1978-267-218	218	morir	to die	281	1494	DIE
+Mountain-1978-267-219	219	matar	to kill	282	1417	KILL
+Mountain-1978-267-220	220	nadar	to swim	283	1439	SWIM
+Mountain-1978-267-221	221	volar	to fly	284	1441	FLY (MOVE THROUGH AIR)
+Mountain-1978-267-222	222	andar	to walk	285	1443	WALK
+Mountain-1978-267-223	223	venir	to come	286	1446	COME
+Mountain-1978-267-224	224	acostarse (acostado)	to lie	287	1411	LIE (REST)
+Mountain-1978-267-225	225	sentarse (sentado)	to sit	288	1416	SIT
+Mountain-1978-267-226	226	ponerse en pie (ponerse de pie)	to stand	289	1442	STAND
+Mountain-1978-267-227	227	dar	to give	290	1447	GIVE
+Mountain-1978-267-228	228	decir	to say	291	1458	SAY
+Mountain-1978-267-229	229	arder	to burn	292	2102	BURN
+Mountain-1978-267-230	230	montaña	mountain	293	639	MOUNTAIN
+Mountain-1978-267-231	231	rojo	red	294	156	RED
+Mountain-1978-267-232	232	verde	green	295	1425	GREEN
+Mountain-1978-267-233	233	amarillo	yellow	296	1424	YELLOW
+Mountain-1978-267-234	234	lleno	full	297	1429	FULL
+Mountain-1978-267-235	235	nuevo	new	298	1231	NEW
+Mountain-1978-267-236	236	seco	dry	300	1398	DRY
+Mountain-1978-267-237	237	nombre	name	301	1405	NAME
+Mountain-1978-267-238	238	mar	sea	311	1474	SEA
+Mountain-1978-267-239	239	lavar	to wash	313	1453	WASH
+Mountain-1978-267-240	240	gusano	worm	314	1219	WORM
+Mountain-1978-267-241	241	ala	wing	315	1257	WING
+Mountain-1978-267-242	242	piel peluda	fur	316	580	FUR
+Mountain-1978-267-243	243	ombligo	navel	317	1838	NAVEL
+Mountain-1978-267-244	244	saliva	saliva	318	1503	SALIVA
+Mountain-1978-267-245	245	leche	milk	319	635	MILK
+Mountain-1978-267-246	246	frotar	to rub	348	1449	RUB
+Mountain-1978-267-247	247	jalar, halar	to pull	349	1455	PULL
+Mountain-1978-267-248	248	empujar	to push	350	1452	PUSH
+Mountain-1978-267-249	249	echar	to throw	351	1456	THROW
+Mountain-1978-267-250	250	golpear	to hit	352	353	STRIKE
+Mountain-1978-267-251	251	hender	to split	353	1437	SPLIT
+Mountain-1978-267-252	252	punzar	to pierce	354	398	PIERCE
+Mountain-1978-267-253	253	cavar	to dig	355	1418	DIG
+Mountain-1978-267-254	254	atar	to tie	356	1917	TIE
+Mountain-1978-267-255	255	coser	to sew	357	1457	SEW
+Mountain-1978-267-256	256	caer	to fall	358	1280	FALL
+Mountain-1978-267-257	257	hinchar	to swell	359	1573	SWELL
+Mountain-1978-267-258	258	pensar	to think	360	1415	THINK (REFLECT)
```

Comment:
THINK in Huber.

Reply:
But THINK (REFLECT) in Morse, which makes use of the same underlying Swadesh/Rowe list if I recall correctly. Based on the Spanish word, which concept would be more appropriate?

Reply:
pensar means both to have an opinion (yo pienso que...) and to reflect (yo pienso en...). THINK is more general, right?

Reply:
That makes sense. I changed it.